### PR TITLE
Add SECURITY.md to show committed PHP versions

### PR DESCRIPTION
**Pull request by vmcj**
Mainly to have the supported PHP versions somewhere. I dont know what we should do with the following questions so I left them open for the case where this happens:

## Reporting a Vulnerability

Use this section to tell people how to report a vulnerability.

Tell them where to go, how often they can expect to get an update on a
reported vulnerability, what to expect if the vulnerability is accepted or
declined, etc.

### CODE REVIEW

**Comment by thijskh** [comment]
Not sure how this relates to security per se. Is it not something more to elaborate on with our system requirements?

**Comment by vmcj** [comment]
> Not sure how this relates to security per se. Is it not something more to elaborate on with our system requirements?

Well it does also list the supported DJ versions, I just added the PHP versions because it was convenient to also have them there.

**Comment by thijskh** [comment]
And I guess we will then not ship this file in the tarball?

**Comment by thijskh** [comment]
I'm not sure about the check mark behind the current development version. Is that actually "supported"?

**Comment by vmcj** [comment]
> I'm not sure about the check mark behind the current development version. Is that actually "supported"?

I think indeed only supporting after releasing is better.

**Review by thijskh — APPROVED** [review]

**Comment by thijskh** [comment]
Or we could put ⚠️ for main. Since I do believe we'd more or less offer support to people using it.

**Comment by vmcj** [comment]
> Or we could put ⚠️ for main. Since I do believe we'd more or less offer support to people using it. And it's a bit in our interest that people do use it if they dare.

Yes, that makes sense also because we will fix security issues in that release.

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+## Supported Versions
+
+| DOMjudge Version | Supported          | PHP version supported |
+| ---------------- | ------------------ | --------------------- |
+| 8.x.x            | :warning:          | 7.4-8.1               |
+| 8.0.x            | :white_check_mark: | 7.2-8.0               |
+| 7.3.x            | :white_check_mark: | 7.2-7.4               |
+| < 7.3            | :x:                | :x:                   |
+
+## Reporting a Vulnerability
```
Comment (meisterT) [inline SECURITY.md:12]: Move this up?

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+If you want to report a vulnerability, please do not use the issue tracker
+or pull request, but instead contact security@domjudge.org so we can assess
```
Comment (meisterT) [inline SECURITY.md:15]: I assume that mailing list / alias exists already?

File: SECURITY.md (hunk 1 above)
Comment (vmcj) [reply SECURITY.md:12]: Its the default template, so I rather keep it default.

File: SECURITY.md (hunk 2 above)
Comment (vmcj) [reply SECURITY.md:15]: According to the domjudge.org server yes,

File: SECURITY.md (hunk 1 above)
Comment (meisterT) [reply SECURITY.md:12]: I didn't know.

**Review by meisterT — APPROVED** [review]

**Pull request merged**